types/node-vault-client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/node-vault-client",
+    "version": "0.0.9999",
```

Comment:
The package is at 1.0.1; this should be `1.0.9999` if it's the types for v1.

Reply:
Thank you @jakebailey. Fixed.